Library/Homebrew/utils/git.rb

```diff
@@ -36,7 +36,7 @@ def last_revision_commit_of_files(repo, files, before_commit: nil)
     out, = Open3.capture3(
       HOMEBREW_SHIMS_PATH/"scm/git", "-C", repo, "log",
       "--pretty=format:%h", "--abbrev=7", "--max-count=1",
-      "--diff-filter=d", "--name-only", *args, "--", *files
+      "--diff-filter=d", "--name-only", *args, "--", files.join(" ")
```

Comment:
Unfortunately, I don't think this works. Since we're not operating within a shell, there's no shell to perform splitting, and git itself isn't treating them as multiple files.

Before:

```
# Git.last_revision_commit_of_files("/usr/local/Homebrew", ["LICENSE.txt", "README.md"])
=> ["ac0665d", ["README.md"]]
```

After:

```
# Git.last_revision_commit_of_files("/usr/local/Homebrew", ["LICENSE.txt", "README.md"])
=> [nil, []]
```